spec/unit/sources_manager_spec.rb

```diff
@@ -42,6 +42,8 @@ module Pod
       @sources_manager = Source::Manager.new(config.repos_dir)
       stub_url_as_cdn('https://cdn.cocoapods.org')
       stub_url_as_cdn('http://cdn.cocoapods.org')
+      stub_url_as_cdn('https://cdn.cocoapods.org/')
```

Comment:
@dnkoutso For some reason the test case that uses `Pod::TrunkSource::TRUNK_REPO_URL` returns `https://cdn.cocoapods.org/` as the URL. This ending slash is fine for local machine but breaks for CI... so I end up adding this and it fixes the issue 🤦 wondering why...

Reply:
cc @igor-makarov is it possible related to the intermittent issue and the change we did? I wonder why we need to have this now.

Reply:
This is possible.

 @gyfelton can you try removing this change and see if itthe build still passes?

Reply:
no it's there to fix an rspec failure on a test case in the first place, let's use the other PR I tagged you at to continue the discussion